src/en/proposals/fairlysadpanda-revs-uplink.md

```diff
@@ -0,0 +1,159 @@
+  - Boring stalemates and unclear round state, especially for the loyalist crew
+  - The endless feels-bad forced round-removal of discovered Head Revs and Heads Of Staff, sometimes for over an hour
+  - Paint-by-numbers round activities entirely focussed on flashes and MindShield implants.
+- Drain on Admins, both due to players demanding ERT revents in response to a discovered revolution and rules enforcement.
+
+Revs is badly in need of a refactor to encourage a more dynamic round that encourages paranoia from Security and smart play from the Revolutionaries. This document forms one half of the "Revs 2.0" refactor, alongside a to-be-written replacement of the Flash/MindShield conversion game mechanics. For the original overview document, [see here](https://github.com/space-wizards/docs/pull/132).
+
+# TL;DR
+
+The current "kill all Heads" win condition for Revolutionaries is replaced.
+
+The bridge Communications Computer is upgraded to a Centcomm Security Uplink terminal.
+
+This terminal is pretty hard to destroy, internally-powered, bolted to the floor and very heavy. If the terminal somehow gets destroyed (a Singuloose, immovable rod, etc) then CentComm will detect this and dispatch the needed machine board to make another to the trade station.
+
+This Computer works like a Communications Computer, but represents the station's connection to CentComm. This connection can be switched off by inserting one of a set of keycards: one of the original Department Head IDs (but not the Captain's spare). This starts a timer. Each additional Head ID reduces this timer. The timer can be disabled by accessing the terminal and cancelling it.
+
+If this timer expires, and it's a Revs round, the Revs win. Even if it isn't, all cards on the station automatically become All Access.
+
+# The Rules
+
+One or more Head Revolutionaries are spawned from the usual pool of players who have opted-in. These Head Revolutionaries have one specific goal: deliver the station, mostly-whole, into Syndicate hands.
+
+- To deliver the station into Syndie hands, the Bridge must be taken over with the **CentComm Uplink Terminal** given at least one Head ID. This triggers a countdown until the uplink is severed and the Revs win. Doing so results in a major syndicate victory and the round ending. Each Head ID input after the first reduces the countdown.
+
+- A draw is achieved if the round ends with no Heads escaping to Centcomm
+
+- A minor crew victory is achieved if the round ends with an evac and at least one Head escapes to Centcomm.
+
+- A major crew victory if all Head Revs are neutralized, either by death, inprisonment or neutralization\*. If this occurs, all Rev members of staff de-convert and Centcomm (belatedly) detects the mind control rays being blasted at the station, sending an evacuation shuttle and allowing the round to end. (If the surviving chain of command wants a longer round, the shuttle can be recalled).
+
```

Comment:
Isn't this still the current issue of heads just leaving or roundstalling.

Reply:
Maybe - I'm happy to workshop the end-of-round situation for crew victories vs revs.

Reply:
I think it's currently fine - Heads hiding leaves room for the revolutionaries to just rush the birdge, giving the Revolutionaries an advantage and Headrevs hiding still results in a crew victory - just a minor one. 

The main one I'm iffy about is the draw condition. Doesn't this encourage round removal of all heads you encounter anyways?

Reply:
On leaving/roundstalling: I'm going to run this past the desgin discussion area of the Discord for views here.

On the draw condition: maybe; I want to give the Revs an opportunity to fight for a draw if Sec completely outplays them, though.

Reply:
That's very fair, I think giving revolutionaries a secondary condition to work towards is a great idea if they fail to get to the bridge. I just don't think it should be killing all heads, as i believe it encourages round removal of heads instead of just killing them for their ID and leaving them there to (maybe) be recovered.

Reply:
Agreed, I'll workshop this.